extensions/blocks/contact-form/editor.scss

```
@@ -192,7 +192,7 @@ input.components-text-control__input {
 
 	// TODO: make this a class, @enej
 	[data-type='jetpack/field-select'] & {
-		border: 1px solid $dark-gray-150;
+		border: 1px solid $gray-200;
```

I wonder if this should be:

```suggestion
		border: 1px solid rgba( 0, 0, 0, 0.4 );
```

To be consistent will other form elements, like: https://github.com/Automattic/jetpack/blob/a234115ceb8146b6e09d00f01c2630bc3b1d305f/extensions/blocks/contact-form/editor.scss#L159